src/api/config/environments/development.rb

```diff
@@ -59,12 +59,3 @@
 
 CONFIG['frontend_host'] = "localhost"
 CONFIG['frontend_protocol'] = "http"
-
-require 'socket'
-fname = "#{Rails.root}/config/environments/development.#{Socket.gethostname}.rb"
-if File.exist? fname
-  STDERR.puts "Using local environment #{fname}"
-  eval File.read(fname)
-else
-  STDERR.puts "Custom development.#{Socket.gethostname}.rb not found - using defaults"
-end
```

Comment:
@adrianschroeter @coolo Would be good if you could double check that this isn't used anymore.

I checked our OBS instance and my personal dev machine and couldn't find a config/development.#{Socket.gethostname}.rb file.

(There is a config/development.sphinx.conf in my vagrant box, but that's for a different purpose, right?)